models/demos/deepseek_v3/tt/mla_1d.py

```diff
@@ -619,6 +621,28 @@ def decode_model_config(
             topology=ttnn.Topology.Linear,
         )
 
+        # Q all-to-all
+        config["wq_a2a_ag"] = AllGatherAsyncConfig(
```

Comment:
nitpick (not necessary): I personally think in-place dictionary construction looks cleaner than building it up piece by piece (like in MLP1D)